### ISSUE TITLE
the esp8266fs file isn't showing up in the tools option

### ISSUE THREAD

**Issue by rohitssj** [esp8266/arduino-esp8266fs-plugin#96]

**Comment by lesleylemmens** [comment]
same problem here. maybe a to new ide?

**Comment by saiaravind19** [comment]
Same here 
I found out that latest release V 0.5.0 is compatible with Arduino IDE version 1.8.5 any later versions the plugin is not working.
Try using the versions mentioned above

**Comment by FlorianHeigl** [comment]
Sam  here, definitely compatibility issue with the new IDE version. installing the older one makes no sense, rather gonna have to move to another SPI driver. currently our wedding pics aren't showing after I just wanted to update the Wifi PSK.
so this is sadz. I hope I can get SPIFFS access without wiping

**Comment by FlorianHeigl** [comment]
this is the upstream issue: https://github.com/arduino/arduino-ide/issues/58

tl;dr
- plugin framework has completely changed. vscode integration improves, java plugins are not grandfathered in.
- all old gui integrated tools aren't working, but expressif will help fix it
- spiffs is deprecated, too, one ought to use one of the two newer libraries (littlefs and fat)
- there is a littlefs plugin https://github.com/earlephilhower/arduino-esp8266littlefs-plugin which also has a gui for arduino >=2. It could be a starting point.

**Comment by earlephilhower** [comment]
Actually, for LittleFS uploads this is the proper IDE 2.x link:  https://github.com/earlephilhower/arduino-littlefs-upload .  It's compatible with the Raspberry Pi Pico and the ESP8266.

I did not make a SPIFFS uploader for 2.x, since SPIFFS is deprecated, but you're free to fork the [littlefs IDE 2.x  plug in](https://github.com/earlephilhower/arduino-littlefs-upload) and do the needful (very straightforward).
